bin/addWelcomePosts.ts

```diff
@@ -14,7 +14,7 @@ import { createSquadWelcomePost } from '../src/common';
 
   resStream.on('data', async (squadSource: SquadSource) => {
     // Fetch first admin based on created data
-    const { userId } = await con.getRepository(SourceMember).findOne({
```

Comment:
Unrelated but we can probably actually just get rid of some bin scripts 😅

Reply:
Agreed